sys-libs/libcxx/files/libcxx-19.1.3-macos-reexports.patch

```diff
@@ -0,0 +1,17 @@
+Fix build of standalone libc++ on Gentoo Prefix
+
+Inspired by:
+https://github.com/llvm/llvm-project/pull/79012
+https://github.com/llvm/llvm-project/pull/110920
```

Comment:
Is this upstream somewhere or does it need to be sent? If the latter, please do so.

Reply:
Click the links :) One of them is how it broke, another one is the fix. But the fix is more involved than this patch, as it's doing more stuff. I'm here partially reverting the breaking change.

Reply:
I did look! But it wasn't clear to me if those were related (hence "inspired") or just a similar type of problem. Please explicitly add that it's essentially a partial revert and that it's fixed on master.

Reply:
Done